Argo/Globals/decode.swift

```diff
@@ -0,0 +1,15 @@
+public func decode<T: JSONDecodable where T == T.DecodedType>(object: AnyObject) -> T? {
+  return decodeResult(object).value
+}
+
+public func decode<T: JSONDecodable where T == T.DecodedType>(object: AnyObject) -> [T]? {
+  return decodeResult(object).value
+}
+
+public func decodeResult<T: JSONDecodable where T == T.DecodedType>(object: AnyObject) -> Parser<T> {
```

Comment:
Couldn't this just be named `decode` as well? If not, it should be `decodeParser` instead of `decodeResult`.